bmv2/Makefile.am

```diff
@@ -147,6 +150,43 @@ if P4THRIFT
     AM_CPPFLAGS += -DP4THRIFT
 endif
 
+# Openflow plugin
+if COND_PLUGIN_OPENFLOW
+unused_for_now = \
```

Comment:
what is the meaning of this? it seems to be the sum of `of_sources` and `of_includes` but I don't see what is the point of it?